src/foam/core/JSON.js

```diff
@@ -523,6 +612,30 @@ foam.LIB({
       useShortNames: false,
       strict: false,
       propertyPredicate: function(o, p) { return ! p.storageTransient; }
+    }),
+
+    // JSON (not FON): Short, but exclude network-transient properties.
+    NetworkJSON: foam.json.JSONParserStringifier.create({
+      pretty: false,
+      formatDatesAsNumbers: true,
+      outputDefaultValues: false,
+      // TODO: No deserialization support for shortnames yet.
+      //      useShortNames: true,
+      useShortNames: false,
+      strict: false,
```

Comment:
(Somewhat duplicated from long comment above). Assuming no bugs in `strict: true` mode, that should work. Do you think that's an acceptable strategy? Perhaps we could then use a subclass to override `foam.json.Strict.parseString()` rather than introducing a new parser/stringifier?